packages/components/src/components/DaCard.vue

```diff
@@ -216,19 +218,31 @@ export default {
   }
 }
 
-.animate-cards {
-  & .card {
+.card--post {
+  transition: all 0.4s ease-out;
+  transform-origin: center;
+}
 
-    & .card__hover {
-      will-change: transform;
-    }
+.card--post::after {
+  content: '';
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  opacity: 0;
+  border-radius: 8px;
+  box-shadow: 0 16px 24px 0 rgba(0, 0, 0, 0.4);
+  transition: opacity 0.4s ease-in-out;
+}
 
-    &:hover,
-    &.hover {
-      & .card__hover {
-        transform: translate(0, -16px) scale(1.02);
-      }
-    }
+.loaded.bright .card--post::after {
```

Comment:
You can simply use `.bright` as a selector

Reply:
Was thinking of the theme and class `.bright` might get used at different point in project, i tried to be a bit careful. Since you mentioned this, it shall be done! Cheers!